packages/insomnia-inso/src/db/adapters/git-adapter.ts

```diff
@@ -1,47 +1,49 @@
-// @flow
 import fs from 'fs';
 import YAML from 'yaml';
 import path from 'path';
 import { emptyDb } from '../index';
 import type { Database, DbAdapter } from '../index';
+import { UNKNOWN } from '../../types';
 
 const gitAdapter: DbAdapter = async (dir, filterTypes) => {
-  dir = path.join(dir, '.insomnia');
+  dir = path.join(dir, '.insomnia'); // Sanity check - do model directories exist?
 
-  // Sanity check - do model directories exist?
   if (!fs.existsSync(path.join(dir, 'Workspace'))) {
     return null;
   }
 
   const db = emptyDb();
 
-  const readAndInsertDoc = async (type: $Keys<Database>, fileName: string): Promise<void> => {
+  const readAndInsertDoc = async (
+    type: keyof Database,
+    fileName: string,
+  ): Promise<void> => {
     // Get contents of each file in type dir and insert into data
     const contents = await fs.promises.readFile(fileName);
-    const obj = YAML.parse(contents.toString());
```

Comment:
I think this was an accidental removal of a semicolon

Reply:
no, flow-to-ts added this (or, actually, I guess this is standard prettier behavior).

basically this is an edge case of javascript syntax.  it's necessary to have the semicolon because the next line begins with parenthesis and to a javascript intrepreter this is ambiguous with a function call.  the parenthesis are necessary now because of the cast.  it is syntactically valid to put the semicolon on the first line but it's widely considered safer to put it on the first character of the next line.  At least that's what I think is going on.

Still, we'll be ok with the semi on the previous line more like it was before so I've switched it back to that.  eventually the parens and the cast shouldn't be needed anyway - is the "big picture" rationale.